addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -69,7 +69,7 @@
                 <span>ExVAT</span>
                 <span>Total</span>
                 <t t-foreach="props.data.tax_details" t-as="tax" t-key="tax.tax.id">
-                    <span t-esc="tax.tax.letter || ''"/>
+                    <span><t t-esc="tax.tax.letter || ''"/> <t t-esc="tax.tax.name"/></span>
```

Comment:
until now we were not planning to show the name of the tax here. Perhaps for this specific l10n where you need the name shown you can just put the name in the `letter` field ?